SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -447,7 +447,13 @@ void USpatialConnectionManager::SetupConnectionConfigFromURL(const FURL& URL, co
 	else
 	{
 		SetConnectionType(ESpatialConnectionType::Receptionist);
-		ReceptionistConfig.SetReceptionistHost(URL.Host);
+
+		// If we have a non-empty host then use this to connect. If not - use the default configured in FReceptionistConfig initialisation.
```

Comment:
@jessicafalk @MatthewSandfordImprobable any chance this change might break existing assumptions?